### PR TITLE
Conways game of life 

### PR DESCRIPTION
**Pull request by LeoraBennett**

### CODE REVIEW

File: src/main/java/GameOfLife.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
+public class GameOfLife {
```
[the remark this answers is not in the document]
Comment (andrewoid) [reply src/main/java/GameOfLife.java]: Your code should be in `bennett.gameoflife`

File: src/main/java/GameOfLife.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        for (int i = -1; i <= 1; i++) {
+            for (int j = -1; j <= 1; j++) {
+                if (i == 0 && j == 0) {
+                    continue;
+                }
+                int newRow = row + i;
+                int newCol = col + j;
+
+                if (newRow >= 0 && newRow < rows && newCol >= 0 && newCol < cols) {
+                    count += grid[newRow][newCol];
```
Comment (andrewoid) [inline src/main/java/GameOfLife.java]: `count++`

File: src/test/java/GameOfLifeTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class GameOfLifeTest {
+
+    @Test
+    public void nextGeneration() {
```
Comment (andrewoid) [inline src/test/java/GameOfLifeTest.java]: `nextGen()`

File: src/main/java/GameOfLife.java (hunk 2 above)
Comment (LeoraBennett) [reply src/main/java/GameOfLife.java]: that increments all of the neighbors whether they are alive or not?

Comment (LeoraBennett) [reply src/main/java/GameOfLife.java]: shouldn't it only be if the neighbor is alive? otherwise the test fails because the output is incorrect

Comment (andrewoid) [reply src/main/java/GameOfLife.java]: Nevermind

File: src/main/java/bennett/gameoflife/GameOfLife.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        for (int i = -1; i <= 1; i++) {
+            for (int j = -1; j <= 1; j++) {
+                if (i == 0 && j == 0) {
+                    continue;
+                }
+                int newRow = row + i;
+                int newCol = col + j;
+
+                if (newRow >= 0 && newRow < rows && newCol >= 0 && newCol < cols) {
+                    count += grid[newRow][newCol];;
```
Comment (andrewoid) [inline src/main/java/bennett/gameoflife/GameOfLife.java]: remove double ;;s

**Review by andrewoid — APPROVED** [review]

**Review by andrewoid — APPROVED** [review]

**Pull request merged**